### PR TITLE
[WIP] Listen to all namespaces

### PR DESCRIPTION
**Pull request by andraxylia**

### CODE REVIEW

File: cmd/manager/main.go [hunk 1]
```diff
@@ -139,6 +140,10 @@ var (
 		Use:   "sidecar",
 		Short: "Istio Proxy sidecar agent",
 		RunE: func(c *cobra.Command, args []string) (err error) {
+			if flags.controllerOptions.Namespace == "" {
+				flags.controllerOptions.Namespace = os.Getenv("POD_NAMESPACE")
```
Comment (kyessenov) [inline cmd/manager/main.go]: Are we disallowing traffic to cross namespaces?

**Comment by kyessenov** [comment]
@andraxylia are you working on this?

**Comment by andraxylia** [comment]
Yes.

**Comment by kyessenov** [comment]
this is likely very out of date. closing it for now, will revisit post-alpha.

**Pull request closed without merging**